examples/distributed/kubernetes/Dockerfile

```diff
@@ -0,0 +1,10 @@
+From python:3.5-slim-buster
```

Comment:
How about making `From` upper case? According to the [docker reference](https://docs.docker.com/engine/reference/builder/), an instruction has better be upper case as the convention.

```
The instruction is not case-sensitive. However, convention is for them to be UPPERCASE to distinguish them from arguments more easily.
```

Reply:
Nice catch. I'll fix it right away